### DISCUSSION TITLE
Is there a way to permanently disable "Recover Unsaved Files" patch...

### DISCUSSION THREAD

**Discussion by JuriTuuti** [phcode-dev/phoenix#1698] [Q&A]
Almost without exception, when I start Phoenix code,  I get a green patch on the top right of the screen, asking me if I want to Recover Unsaved Files.  

I am using Phoenix code in Windows 10.

I cannot escape it, I will need to take the mouse, and select discard or restore.  I have tried to save all files before I exit the program, and close all working windows.  It still appears.  It does not seem to have any sensible purpose, since there are no unsaved files that I would want to recover.   The only thing it seems to do is to add a file-has-changed -dot in front of the just opened working files -- even though there was none when the program was last closed.

I tried to install an older version to see if I can get rid of it, but it is the same all the way down to 3.4.4

**Comment by abose** [comment]
Thanks for reporting the issue. Lete investigate, will fix in june release.
